samples/loadimpactair_script_sample.js

```diff
@@ -0,0 +1,403 @@
+/*
+ *
+ * This script is a user journey for http://loadimpactair.guldskeden.se
+ * Copyright (C) 2017 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+import { group, sleep } from "k6";
+import http from "k6/http";
+import { Trend } from "k6/metrics";
+
+/**
+ * Returns random number between (inclusive) min and max
+ */
+function rnd(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+/**
+ * Returns URL encoded string
+ */
+function urlencode (str) {
+
+  str = (str + '')
+
+  return encodeURIComponent(str)
+    .replace(/!/g, '%21')
+    .replace(/'/g, '%27')
+    .replace(/\(/g, '%28')
+    .replace(/\)/g, '%29')
+    .replace(/\*/g, '%2A')
+    .replace(/%20/g, '+')
+}
+
+/**
+ * Returns __VIEWSTATE found in body or empty string if not found
+ */
+function findViewstate(body) {
+	var strViewstate = "";
+    var findVS = new RegExp('id="__VIEWSTATE" value="(.*?)"' )

```

Comment:
Why not use `k6/html` to do this instead of p̳̟͔̩͚̜a͚͍̻̫̟ͅr̘s̢i̛͎ͅn̻̺͖̭͕̝͙g̼̗̬ ́H̤͖̬͇Ț̴̟̖͎M̝̘̣̜̥L͎̘͙̫̗͔ ͙̣w̹̞̪i̘̞̮̼̫t͞h̨̟̩͈̭͍ ̘̲̀r̫̦̣e̫̝͝g̘͞ex̷̩?